hedera-node/configuration/previewnet/application.properties

```diff
@@ -1 +1,4 @@
-
+bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
+contracts.chainId=297
+ledger.id=0x02
+contracts.evm.version.dynamic=true
```

Comment:
I wonder if there should be a similar comment as in the beginning of the testnet version of this file?  Also, is there a mainnet version of this file yet?